### PR TITLE
Sort songs

### PR DESCRIPTION
**Pull request by gracewainaina**
sort songs in playlist based on the number of likes

### CODE REVIEW

**Comment by gracewainaina** [comment]
Here is a demo showing how songs are sorted and pull to refresh to update the ranking of songs based on the number of likes: https://github.com/gracewainaina/Musicca/blob/sort-songs/Screenshots/swipe_to_refresh_sortedsongs.mp4

File: Musicca/app/src/main/java/com/example/musicca/adapters/CurrentPlaylistAdapter.java [hunk 1]
```diff
@@ -45,6 +47,7 @@
 
     private static final String TAG = "Queue";
 
+    public List<String> sortedSongObjectIds;
```
Comment (xuyueee) [inline Musicca/app/src/main/java/com/example/musicca/adapters/CurrentPlaylistAdapter.java]: Why does this need to be public? In general it's discouraged to expose the members directly. You can create getter methods for them.

File: Musicca/app/src/main/java/com/example/musicca/adapters/CurrentPlaylistAdapter.java [hunk 2]
```diff
@@ -45,6 +47,7 @@
...
     private Context context;
     private List<String> songObjectIds;
```
Comment (xuyueee) [inline Musicca/app/src/main/java/com/example/musicca/adapters/CurrentPlaylistAdapter.java:52]: Do you still need two list of songs, one list sorted, the other unsorted? Can you keep the list always sorted?

Comment (gracewainaina) [reply Musicca/app/src/main/java/com/example/musicca/adapters/CurrentPlaylistAdapter.java:52]: The unsorted list is passed into the adapter from the activity and then I use the unsorted list to build the comparable model that helps me populate the sorted list.

**Pull request merged**